content/blog/crosswalk-for-kubernetes-intro/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Introducing Crosswalk for Kubernetes"
```

Comment:
Why is this post included in the .NET blog post PR?

Reply:
I put it in the same branch, will create a separate branch